blueprints/cloud-operations/network-dashboard/cloud-function/metrics/subnets.py

```diff
@@ -134,7 +139,7 @@ def compute_subnet_utilization(config, all_subnets_dict):
     address = ''
     for versioned in asset.versioned_resources:
       for field_name, field_value in versioned.resource.items():
-        if 'loadBalancingScheme' in field_name and field_value == 'INTERNAL':
+        if 'loadBalancingScheme' in field_name and field_value == 'INTERNAL' or field_value == 'INTERNAL_MANAGED':
```

Comment:
```suggestion
        if 'loadBalancingScheme' in field_name and field_value in ["INTERNAL", "INTERNAL_MANAGED"]':
```

For more clear and better readable and extensible approach